ynr/apps/moderation_queue/review_required_helper.py

```diff
@@ -216,6 +217,25 @@ def needs_review(self):
             return self.Status.UNDECIDED
 
 
+class RevertedEditByAnotherUser(BaseReviewRequiredDecider):
+    def review_description_text(self):
+        return "Edit reverted by another user"
+
+    def needs_review(self):
+        from candidates.models import LoggedAction
+
+        if self.logged_action.action_type == "person-revert":
+            recent_revert_qs = LoggedAction.objects.filter(
+                person=self.logged_action.person,
+                action_type="person-revert",
+                # updated in the last 24 hours
+                updated__gt=datetime.now() - timedelta(hours=24),
+            ).order_by("updated")
+            if recent_revert_qs.count() >= 2:
```

Comment:
This works, but might make it hard to tweak later. Can you pull the values (`24` hours and `2` edits) in to https://github.com/DemocracyClub/yournextrepresentative/blob/master/ynr/settings/constants/needs_review.py?